client/src/components/chatComponents/displayMessages/DisplayMessages.jsx

```diff
@@ -0,0 +1,57 @@
+import "./displayMessages.css";
+import { useEffect, useRef } from "react";
+import SingleChat from "./singleChat/SingleChat.jsx";
+
+function DisplayMessages() {
+  const fieldRef = useRef(null);
+
+  useEffect(() => {
+    fieldRef.current.scrollIntoView();
+  });
+
+  return (
+    <div className="chat-box">
+      <SingleChat
+        whosChat={"sender"}
+        sentAt={"6:54 AM"}
+        messageText={"My name a jeff"}
+      />
+      <SingleChat
+        whosChat={"receiver"}
+        sentAt={"7:10 AM"}
+        messageText={"And my name a john LOLLLL"}
+      />
+      <SingleChat
```

Comment:
Will be good to factor dummy conversation out into a dummy_dict as it will more closely follow the format we will use it in.